testing_and_setup/compass/ocean/hurricane/USDEQU240at60cr20rr4WD/sandy/config_forward_RK4.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0"?>
+<config case="forward_RK4">
```

Comment:
@pwolfram , I don't know what you think, but this could probably just be named `forward`. I don't think the distinction is necessary at this point since WD is restricted to RK4, i.e. there isn't be an equivalent SE case right now.

Reply:
I'm split here-- long term we want to have SE which is why I left it in for now.

Reply:
I see. If that's the case, then let's leave it.